react-toggle-switch/Toggle-Switch/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [onOff, setOnOff] = useState(false);
```

Comment:
The problem with this name is that it's not clear what a value of `true` is. Does it mean the switch is on? Or off?